bts_phot/calibrate_fps.py

```diff
@@ -331,6 +331,8 @@ def get_baseline(fps_file, window="10D",
                                           ) <= 5)
                     if len(mask[0]) > 2:
                         ##### WARNING WARNING WARNING THIS SHOULD BE MEDIAN
+                        # Cmean still reported, using median for baseline - AW
```

Comment:
You can delete this comment and the line above and resubmit the PR

Reply:
Yes, that is correct.